config/config.go

```diff
@@ -0,0 +1,20 @@
+// Copyright 2018 RedHat
```

Comment:
2019 :-)

Reply:
haha! All the others probably say 2018 too!